src/Cache/BlockSsiCache.php

```diff
@@ -47,16 +50,28 @@ class BlockSsiCache extends SsiCache
      */
     protected $contextManager;
 
+    /**
+     * @var ControllerResolverInterface
+     */
+    protected $resolver;
```

Comment:
```suggestion
    private $resolver;
```

Reply:
Does not look like this one is resolved

Reply:
If I change that the level access, there are a problem with parent class